bigbluebutton-html5/imports/ui/components/whiteboard/whiteboard-toolbar/styles.scss

```diff
@@ -3,16 +3,16 @@
 :root {
   --toolbar-margin: .8rem;
   --toolbar-box-shadow: 0 0 10px -2px rgba(0, 0, 0, .25);
-  --toolbar-button-color: var(--btn-default-color);
+  --toolbar-button-color: #5594D4;
```

Comment:
Can we add this color to btn-default-color scss file so we can continue using that variable